lib/class-wp-rest-url-details-controller.php

```diff
@@ -320,11 +315,26 @@ private function get_image( $meta_elements, $url ) {
 		return $image;
 	}
 
+	/**
+	 * Prepare the metadata by:
+	 *
+	 *    - stripping all HTML tags and tag entities
+	 *    - converting non-tag entities into characters.
+	 *
+	 * @param string $metadata The metadata content to prepare.
+	 * @return string The prepared metadata.
+	 */
+	private function prepare_metadata_for_output( $metadata ) {
+		$metadata = html_entity_decode( $metadata, ENT_QUOTES, get_bloginfo( 'charset' ) );
+		$metadata = wp_strip_all_tags( $metadata );
+		return $metadata;
+	}
```

Comment:
@getdave Changed this helper method to do the following:

- strips all HTML tags and tag entities
- converts non-tag entities into characters

Updated the test data to validate.

With this change, the API will expose metadata without entities and HTML. Clean and safe.